internal/app/api/handler/login/login.go

```diff
@@ -50,3 +65,101 @@ func (h *LoginHandler) GetUser(w http.ResponseWriter, r *http.Request) {
 	respond.JSON(w, http.StatusAccepted, u)
 	return
 }
+func (h *LoginHandler) Register(w http.ResponseWriter, r *http.Request) {
+	username := r.FormValue("username")
+	rawPassword := r.FormValue("password")
+	fullName := r.FormValue("fullname")
+	email := r.FormValue("email")
+	deliveryCenter := r.FormValue("deliverycenter")
+	dcs := []string{deliveryCenter}
+	bytes, err := bcrypt.GenerateFromPassword([]byte(rawPassword), 14)
+	if err != nil {
+		h.logger.Errorf("password encryption is fail err: %v", err)
+		respond.JSON(w, http.StatusUnauthorized, "register fail")
+		return
+	}
+	password := string(bytes)
+	user := types.User{
+		ID:             bson.NewObjectId(),
+		Username:       username,
+		FullName:       fullName,
+		Email:          email,
+		DeliveryCenter: dcs,
+		Password:       password,
+	}
+	err = h.srv.AddUser(&user)
+	if err != nil {
+		h.logger.Infof("can't save new user into database err: %v ", err)
+		respond.JSON(w, http.StatusInternalServerError, "add user fail")
+		return
+	}
+	respond.JSON(w, http.StatusCreated, "save user successfully")
```

Comment:
Imagine that you are writing the UI, you have just created a user and you want to jump into that user's profile for viewing/adding more information. How can you do that with the result from backend as "save user successfully"?